scripts/base/DeployValidlyBase.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {Script} from 'forge-std/Script.sol';
+import 'forge-std/StdJson.sol';
+import 'forge-std/console.sol';
+
+import {SovereignPoolFactory} from "@valantis-core/pools/factories/SovereignPoolFactory.sol";
+import {SovereignPool} from "@valantis-core/pools/SovereignPool.sol";
+import {ProtocolFactory} from "@valantis-core/protocol-factory/ProtocolFactory.sol";
+import {Validly} from "src/Validly.sol";
+import {ValidlyFactory} from "src/ValidlyFactory.sol";
+
+abstract contract DeployValidlyBase is Script {
+  using stdJson for string;
+
+  ValidlyFactory public factory;
+  ProtocolFactory protocolFactory;
+  SovereignPoolFactory poolFactory;
+
+  function _deployFactory() public {
+    protocolFactory = new ProtocolFactory(vm.envAddress('SENDER'));
+
+    poolFactory = new SovereignPoolFactory();
+
+    protocolFactory.setSovereignPoolFactory(address(poolFactory));
+
+    // Create ValidlyFactory
+    factory = new ValidlyFactory(address(protocolFactory), 1);
```

Comment:
This will always deploy a ValidlyFactory with 1 bips fee tier. Is this intentional, even for volatile pairs? If not, consider making it a function argument

Reply:
Okay I see a new factory is required for each new fee tier.